app/views/small_grants_fund/index.html.erb

```diff
@@ -160,15 +160,15 @@
       <div class="m-card -fullWidth">
         <h2 class="card-title">Resources and support</h2>
 
-        <a href="" class="card-title-subitle _bold _uppercase">small grand funds webinars</a>
+        <a href="/getinvolved/apply-to-the-small-grants-fund" class="card-title-subtitle _bold _uppercase">small grand funds webinars</a>
```

Comment:
This is incorrect. It should point to the new howto, As long as we are deploying it, I will change it, don't worry